packages/mui-material/src/Select/SelectInput.js

```diff
@@ -417,7 +417,11 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
   }
 
   if (computeDisplay) {
-    display = multiple ? displayMultiple.join(', ') : displaySingle;
+    if (multiple) {
+      display = displayMultiple.reduce((prev, curr) => [prev, ', ', curr]);
```

Comment:
By doing this, the rendered output will be the react element. with:

```js
<Select multiple value={[10, 20, 30]}>
    <MenuItem value={10}>Ten</MenuItem>
    <MenuItem value={20}>
      <strong>Twenty</strong>
    </MenuItem>
    <MenuItem value={30}>Thirty</MenuItem>
  </Select>
```

what you see in the selection will be "Ten, **Twenty**, Thirty". Should it be like this?

Reply:
As can be seen in [here](https://codesandbox.io/s/eager-banach-493dv?file=/src/App.js), we already render react elements in case of `Select` with `multiple` disabled. I think we should keep the consistency. Otherwise, it looks like a bug.

Reply:
Okay, then this looks good to me.

Reply:
The code is missing an initial state param on the reduce which is throwing browser errors.